Source/WebKit/Scripts/generate-serializers.py

```diff
@@ -824,7 +805,7 @@ def parse_serialized_types(file, file_name):
                 serialized_enums.append(SerializedEnum(namespace, name, underlying_type, members, type_condition, attributes))
             else:
                 serialized_types.append(SerializedType(struct_or_class, namespace, name, parent_class_name, members, type_condition, attributes, metadata))
-                if namespace is not None and (attributes is None or 'CustomHeader' not in attributes and 'Nested' not in attributes and 'NoForwardDeclaration' not in attributes):
+                if namespace is not None and (attributes is None or 'CustomHeader' not in attributes and 'Nested' not in attributes):
```

Comment:
Should this be `if namespace is not None and ((attributes is None) or ('CustomHeader' not in attributes and 'Nested' not in attributes)):`

Reply:
Yeah.